store/todos.js

```diff
@@ -124,4 +139,33 @@ export const mutations = {
     state.perPage = 9;
     state.page = 1;
   },
+  setListPerPage(state, val) {
+    state.taskListPerPage = state.taskList;
+  },
+  filterTaskList(state, val) {
+    state.filterOptions.map((option) => (option.status = false));
+    const option = state.filterOptions.find((option) => option.id === val.id);
+
+    option.status = true;
+
+    if (val.title === 'All') {
```

Comment:
Please use a CONST here. 
```suggestion
    import { ALL, INCOMPLETE, COMPLETE } from "@/utils/constants.js"

    if (val.title === ALL) {
```

Reply:
Resolved